Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "versionize_derive"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Hmm, do these first two commits just cancel each other out?

Reply:
This look like commits that I have rebased on top of. I did do a pull rebase, however my local git history does not show it.